### PR TITLE
Update index examples and README

### LINKED ISSUE #30: 🚀 Release 0.1.0 (H_e) 📦
Master issue for the first release of ipyradiant. 

TODO 
* [x] merge pull#29
* [x] merge pull#23
* [x] Update screencast @zwelz3 (pull#32)
* [x] tag
* [x] pip release @nrbgt 
* [x] conda release @bollwyvl
* [x] update readme with install instructions, badges #39

### PR DESCRIPTION
**Pull request by zwelz3**
* [X] Create new screencasts
* [X] Update README with screencasts and install instructions
[Binder](https://mybinder.org/v2/gh/zwelz3/ipyradiant/gh-30-update-index-examples?urlpath=lab/tree/index.ipynb)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,14 +12,23 @@ the Jupyter notebook.
...
+ ![TabApp screencast](https://user-images.githubusercontent.com/32652349/90517340-44148a80-e133-11ea-9ee4-add09ee2c0d4.gif)
 
+
+## Installation
 ```bash
-# TBD
-# pip install ipyradiant
-# jupyter labextension install jupyter-radiant
+pip install ipyradiant
+jupyter labextension install @jupyrdf/jupyter-radiant
```
Comment (nrbgt) [inline README.md]: actually, there is no npm package, but you do want all the dependencies:

```bash
jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-cytoscape @pyviz/jupyterlab_pyviz
```

Comment (zwelz3) [reply README.md]: This should resolve bc917b1

File: README.md [hunk 2]
```diff
@@ -12,14 +12,23 @@ the Jupyter notebook.
   https://github.com/jupyrdf/ipyradiant/workflows/.github/workflows/ciV0.yml/badge.svg
 [status-link]: https://github.com/jupyrdf/ipyradiant/actions
 
-![radiant screencast](https://user-images.githubusercontent.com/32652349/88299868-db232980-ccd0-11ea-9fc8-f21a1047db81.gif)
```
Comment (nrbgt) [inline README.md:15]: we should also be repping `datashader`, `holoviews` and `ipyctoscape`.

File: README.md [hunk 3]
```diff
@@ -12,14 +12,23 @@ the Jupyter notebook.
...
+![datashader screencast](https://user-images.githubusercontent.com/32652349/90517352-470f7b00-e133-11ea-8cb8-8e810198ced0.gif)
+
 
-> For now, see [CONTRIBUTING.md](./CONTRIBUTING.md)
+## Example Tooling Widgets
+`ipyradiant` includes examples where visualization and utility widgets are linked into example tooling.
+ ![TabApp screencast](https://user-images.githubusercontent.com/32652349/90517340-44148a80-e133-11ea-9ee4-add09ee2c0d4.gif)
 
+
+## Installation
```
Comment (nrbgt) [inline README.md:36]: also, probably add someting like 
```md
## Prerequisites

- `python >=3.6`

`ipyradiant`'s python dependencies will install their required `nbextensions` for Notebook Classic

For JupyterLab support, ensure you have the following installed:
- `jupyterlab >=1`
- `nodejs >=10`
```

File: README.md (hunk 2 above)
Comment (zwelz3) [reply README.md:15]: Do you have a suggestion on the best way to do that?

Comment (nrbgt) [reply README.md:15]: `Powered by [][ipyctoscape], [][datashader], and [][holoviews]`

then have links

File: README.md (hunk 3 above)
Comment (zwelz3) [reply README.md:36]: 0b78456

File: README.md (hunk 2 above)
Comment (zwelz3) [reply README.md:15]: 9fc0a1e

**Comment by nrbgt** [comment]
Looking good... perhaps we update/link to the architecture diagram as well?

I'm finding a raft of stuff to tweak on https://github.com/jupyrdf/ipyelk/pull/11 .... will be some lessons learned to bring over here.

**Comment by nrbgt** [comment]
ol' winders is probably going to time out here (and on #33)... perhaps we probably need a smaller test artifact for use in CI

**Comment by nrbgt** [comment]
well lookie there! approvin', can iterate on #33

File: README.md [hunk 4]
```diff
@@ -12,14 +12,23 @@ the Jupyter notebook.
...
 ```bash
-# TBD
-# pip install ipyradiant
-# jupyter labextension install jupyter-radiant
+pip install ipyradiant
+jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-cytoscape @pyviz/jupyterlab_pyviz
```
Comment (nrbgt) [inline README.md]: oops, still missing `qgrid2` here...

Comment (zwelz3) [reply README.md]: Added. db7a9ae

**Review by nrbgt — APPROVED** [review]
GTG... we can iterate on #33, etc

**Pull request merged**